.gitignore

```diff
@@ -1,5 +1,6 @@
 /target/
 /.idea/
+.vscode/
```

Comment:
This is to exclude configuration by my editor.

Reply:
LGTM